pandas/core/window.py

```diff
@@ -963,8 +963,20 @@ def count(self):
         not passed. In the future `raw` will default to False.
 
         .. versionadded:: 0.23.0
+    *args, **kwargs :
+        Arguments and keyword arguments to be passed into func.
 
-    \*args and \*\*kwargs are passed to the function""")
+    Returns
+    -------
+    Series or DataFrame
+        Returned object type is determined by the caller of the
```

Comment:
The last few words here are kind of strange; I think it's OK to just say `Return type is determined by the caller.` as I think we have that exact verbiage in other places

Reply:
Awesome thanks @WillAyd, changes have been pushed.